### ISSUE TITLE
Feature request: shared definitions for all declarations

### ISSUE THREAD

**Issue by dvogel** [xntrik/hcltm#61]
In many organizations (especially larger ones) multiple threat models may interact with the same external assets, rely on the same information assets, and face similar threats. For example, large corporations will often setup shared service teams for common application deployment concerns. A common example is logging to an IT-managed resource like Splunk.

To enable ease of maintenance for threat models in these organizations, I would like to propose generic inheritance feature whereby partial threat models can be defined in one file and grafted into other threat models. This could be recursive but would be very useful even if it only worked one level deep.

```hcl
# this would be in shared/splunk-logging.hcl
threatmodel "Splunk Logging Assets" {
  information_asset "Splunk Logs" {
    description = "The central corporate splunk log storage and search service."
    information_classification = "Confidential"
  }
}
```

```hcl
threatmodel "Corporate App 1" {
  including "shared/splunk-logging.hcl"
  
  threat {
    description = "Our application includes a dynamic logging level that can be changed by IT during debugging scenarios. During periods of higher logging verbosity the application logs the JWT provided by the API caller."
    control = "Splunk ingestion filtering rules."
  }
}
```

In this example, the threat model "Corporate App 1" would be parsed. Then any the threat models named by any `including` stanzas would be parsed. In this case that would be "Splunk Logging Assets". A composite threat model would then be constructed by merging each, with the last defined winning any name collisions.

**Comment by xntrik** [comment]
This is slick! We have _some_ importing (for example here: https://github.com/xntrik/hcltm/blob/14cbe0aeb9b72b99ba2d9d24d7a86bff2174cced/spec.hcl#L143-L165) but I like your suggestion a lot!

**Comment by xntrik** [comment]
Done ! available in v0.1.4
